### DISCUSSION TITLE
Pull Request dashboard

### DISCUSSION THREAD

**Discussion by gbraad** [crc-org/crc#2280] [Show and tell]
I have been busy with a dashboard that runs as a webpage instead of a server application. This makes use and deployment a while lot easier.

I created a repo with some issues here: https://github.com/gbraad/github-dashboard and deployed with github pages at https://gbraad.nl/github-dashboard

![Ez-6GeCVUAAM_Yt](https://user-images.githubusercontent.com/1894/116339184-91dba980-a80f-11eb-98f4-ccdf58d6cadc.jpeg)

**Comment by cfergeau** [comment]
Maybe this could be hosted on `https://crc.dev` somehow ;)

**Reply by gbraad** [reply]
This shouldn't be too difficult. We can create a user account for `crc-team`, with a GitHub Pages repo as `[username].github.io` with an associated `CNAME` file for `crc.dev`. Any repo that is then created as part of this user can have a Pages branch published as `crc.dev/[reponame]`. This is how I host it now ;-)

**Reply by praveenkumar** [reply]
> Maybe this could be hosted on `https://crc.dev` somehow ;)

you mean along with docs or just this instead of doc? :)

**Reply by anjannath** [reply]
> This shouldn't be too difficult. We can create a user account for crc-team, with a GitHub Pages repo as 

We have a github account crc-bot that we can re-use, currently we use its tokens to push to the docs to gh-pages in the crc repo

**Reply by cfergeau** [reply]
> > Maybe this could be hosted on `https://crc.dev` somehow ;)
> 
> you mean along with docs or just this instead of doc? :)

Ideally both can be there. And links to our releases/release notes/...

**Reply by gbraad** [reply]
Should we host this on crc.dev? Easy to do by adding a ghpage for this repo under the crc-org.

**Reply by anjannath** [reply]
Yes, would be useful i think
